### DISCUSSION TITLE
3.4.0 version release

### DISCUSSION THREAD

**Discussion by pockyhe** [argoproj/argo-workflows#9390] [Q&A] [answered]
Hi, I ‘m looking forward to the release of 3.4.0 version. We wanna enable Azure Storage Blob feature in k8s by installing helm chart. However, Argo-Helm now just support Argo workflow 3.3.9 which didn't include this new feature. We really hope to know your plan about releasing 3.4.0 for Argo Workflows and when the Argo Helm Chart will include this version. Thank you very much

**Comment by terrytangyuan** [comment, marked as answer]
We don’t maintain those Helm charts. Please file an issue there instead: https://github.com/argoproj/argo-helm

**Reply by terrytangyuan** [reply]
cc Based on the CODEOWNERS file: @stefansedich @paguos @vladlosev @yann-soubeyrand @jmeridth @yu-croco

https://github.com/argoproj/argo-helm/blob/main/CODEOWNERS

**Reply by pockyhe** [reply]
But I believe it depends on the released version of Argo Workflows. So we really hope to know your plan about releasing 3.4.0 for Argo Workflow

**Comment by jmeridth** [comment]
@pockyhe thank you for asking and starting the [discussion](https://github.com/argoproj/argo-helm/discussions/1406) over on the argo-helm repo.  Thank you @terrytangyuan for the ping.  Cheers.
